index.html

```diff
@@ -332,6 +332,27 @@ <h3>Let's connect</h3>
 
             </section>
         </section>
+        <section class="contact-form-container">
+            <section class="contact-form">
+                <h2>Contact Me</h2>
+
+                <p>If you have an application you are interested in developing, a
+                    feature that you need built or a project that needs coding.
+                    I’d love to help with it!</p>
+                  <form action="https://formspree.io/f/xoqzgvow" method="post"
+                    enctype="multipart/form-data">
+                    <input type="text" placeholder="Lorem ipsum"
+                      maxlength="30"
+                      required>
+                    <input type="email" name="email" placeholder="loremispum@gmail.com"
+                      required>
+                    <textarea rows="7" name="message"
+                      placeholder="Write your message here..." maxlength="500"></textarea>
```

Comment:
> A user cannot submit the form if one or more of the fields is empty.
- [ ] As stated in the project requirement, all input fields need to be filled before a user can submit a form. Currently, that's not the case in your project as your `textarea` doesn't have the required attribute. I'll suggest you add the `required` attribute to the textarea in order to meet this requirement. :+1: :wave: